__tests__/traders/LimitOrders.test.ts

```diff
@@ -115,6 +115,11 @@ describe('LimitOrders', () => {
     });
 
     it('Succeeds for eth_signTypedData', async () => {
+      // TODO: remove conditional when ethereumjs-testrpc-sc supports arrays in eth_signTypedData
```

Comment:
solidity-coverage uses a fork of ganache, so it doesn't have support for eth_signTypedData for arrays yet